contracts/src/v0.8/vrf/dev/VRFCoordinatorV2_5.sol

```diff
@@ -44,11 +44,11 @@ contract VRFCoordinatorV2_5 is VRF, SubscriptionAPI, IVRFCoordinatorV2Plus {
     address sender;
     bytes extraArgs;
   }
-  mapping(bytes32 => address) /* keyHash */ /* oracle */ public s_provingKeys;
+  mapping(bytes32 => bool) /* keyHash */ /* exists */ public s_provingKeys;
   bytes32[] public s_provingKeyHashes;
   mapping(uint256 => bytes32) /* requestID */ /* commitment */ public s_requestCommitments;
-  event ProvingKeyRegistered(bytes32 keyHash, address indexed oracle);
-  event ProvingKeyDeregistered(bytes32 keyHash, address indexed oracle);
+  event ProvingKeyRegistered(bytes32 keyHash);
+  event ProvingKeyDeregistered(bytes32 keyHash);
```

Comment:
Now, proving keys are not registered to an oracle?

Reply:
yes correct

Reply:
So, no matter how many proving keys we register, all earnings can only be withdrawn by contract owner now